demos/default/pages/deep/import.js

```diff
@@ -20,9 +20,7 @@ const Show = ({ show }) => (
 
     <hr />
 
-    <Link href="/">
-      <a>Go back home</a>
-    </Link>
+    <Link href="/">Go back home</Link>
```

Comment:
For context, you no longer need to do this.

> Note: Before Next.js 12.2, it was required that the Link component wrapped an `<a>` tag, but [this is not required in versions 12.2 and above](https://nextjs.org/blog/next-12-2).

See https://nextjs.org/learn/basics/navigate-between-pages/link-component